qa/rolling-upgrade/src/test/resources/rest-api-spec/test/old_cluster/10_basic.yaml

```diff
@@ -50,6 +50,18 @@
       indices.flush:
         index: test_index,index_with_replicas
 
+---
+teardown:
+  - do:
+      indices.delete:
+          index: test_index
```

Comment:
I don't think this is right. I think this'll break the tests for the mixed version.

I wonder if a better solution is use a different index to test the search template rather than the one that we use for testing the rest of the stuff.